app/routes/authentication.py

```diff
@@ -96,6 +117,43 @@ def register_user():
     return "", 204
 
 
+@auth_blueprint.route("/user/<int:user_id>", methods=["PATCH"])
+@fresh_jwt_required
+def update_user(user_id):
+    if not user_id:
+        return json_api(BadRequest, ErrorSchema), 400
+
+    current_user = User.query.filter_by(email=get_jwt_identity()).first()
+    user = User.query.filter_by(id=current_user.id).first_or_404()
+
+    try:
+        json_schema_manager.validate(request.json, "user.json")
+        new_email = dig(request.json, "email", None)
```

Comment:
So - pointing out that the `dig` method is mostly useful for querying deeply nested JSON - you wouldn't need it in this case - as this is equivalent to `request.json["email"]`

That said I'm not apposed to using "dig" as a matter of principle on this kind of thing since it makes all of the code paths similar.